github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-presubmits.yaml

```diff
@@ -491,7 +491,7 @@ presubmits:
         - /usr/local/bin/runner.sh
         - /bin/sh
         - -c
-        - cd cluster-provision/k8s/1.26 && ../provision.sh
+        - cd cluster-provision/k8s/1.26 && KUBEVIRT_PSA='true' ../provision.sh
```

Comment:
Sorry for this late remark, but I think you should do this in a separate PR, since we can't change the kubevirt/kubevirt lanes to enable psa before the psa is working inside kubevirt.

Reply:
Gotcha, removed it!

Reply:
Thanks! LMK when you have created the other PR.

@brianmcarey I think we might set the psa check-provisions to `always_run: true` so we get any failures. WDYT?

Reply:
Yes I think we should have it set to always run. It is an important provider for kubevirt.